git_clases.rb

```diff
@@ -89,9 +81,10 @@ def post(uri, request)
             puts "Responde Si o No..."
         end
 
-        open(filename, "r") { |file| @contenido = file.read() } 
-        content = @contenido
-        uri = URI.parse("https://api.github.com/gists")
+        open(filename, "r") { |file| @content_file = file.read() } 
+        content = @content_file
+        GITHUB_URL = "https://api.github.com/gists"
```

Comment:
It would be better to add the constant definition at the top of the class, that way it's clearly visible.